binary_gentoo/internal/cli/tree_diff.py

```diff
@@ -22,11 +82,23 @@ def iterate_new_and_changed_ebuilds(config):
         for ebuild_file in ebuild_files:
             old_portdir_ebuild_file = os.path.join(config.old_portdir, category_plus_package,
                                                    ebuild_file)
+            new_portdir_ebuild_file = os.path.join(root, ebuild_file)
+
+            # don't output 9999 ebuilds
+            if ebuild_file.endswith('-9999.ebuild'):
```

Comment:
OK. There are indeed many other cases where 9999 is included. But I noticed it is possible to have a revision as well, so I'll make a regex out of it to be complete:
```
ci-visualization/gnuplot/gnuplot-5.5.9999-r1.ebuild
dev-libs/gnulib/gnulib-9999-r1.ebuild
dev-vcs/git/git-9999-r1.ebuild
dev-vcs/git/git-9999-r3.ebuild
dev-vcs/git/git-9999-r2.ebuild
```
Just plain filtering out if "9999" is included in the version number is not an option, because there are a few packages that probably should not be filtered out that have 9999 in their version:
```
dev-ruby/ruby-odbc/ruby-odbc-0.999991.ebuild
portdir/dev-perl/File-Slurp/File-Slurp-9999.270.0.ebuild
portdir/dev-perl/File-Slurp/File-Slurp-9999.320.0.ebuild
```

Reply:
Good catch!